Library/ViewModels/PledgeCTAContainerViewViewModel.swift

```diff
@@ -47,7 +50,7 @@ public final class PledgeCTAContainerViewViewModel: PledgeCTAContainerViewViewMo
           basicPledge,
           country: project.country,
           omitCurrencyCode: project.stats.omitUSCurrencyCode
-        )
+        ).trimmed()
```

Comment:
This should probably be tested in our VM to make sure that the output is indeed trimmed. Current tests need better template values since those provided values are always trimmed.

Reply:
We can pair on this